src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -1,12 +1,14 @@
 package com.minecolonies.coremod.util;
 
 import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.blocks.BlockTimberFrame;
 import com.minecolonies.coremod.blocks.ModBlocks;
 import com.minecolonies.coremod.items.ModItems;
 import net.minecraft.block.Block;
 import net.minecraft.init.Blocks;
 import net.minecraft.init.Items;
 import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.item.ItemBlock'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)